db/db_impl_secondary.h

```diff
@@ -194,19 +194,25 @@ class DBImplSecondary : public DBImpl {
 
   using DBImpl::Recover;
 
-  Status FindAndRecoverLogFiles();
+  Status FindAndRecoverLogFiles(
+      std::unordered_set<ColumnFamilyData*>* cfds_changed,
+      JobContext* job_context);
   Status FindNewLogNumbers(std::vector<uint64_t>* logs);
   Status RecoverLogFiles(const std::vector<uint64_t>& log_numbers,
```

Comment:
since we no longer override `RecoverLogFiles` from DBImpl, do you consider slightly renaming the function to avoid confusion?

Reply:
I think it's OK to use function overloading here.